module/Application/Module.php

```diff
@@ -25,12 +25,12 @@ public function getConfig()
 
     public function getAutoloaderConfig()
     {
-        return array(
-            'Zend\Loader\StandardAutoloader' => array(
-                'namespaces' => array(
+        return [
+            'Zend\Loader\StandardAutoloader' => [
```

Comment:
Actually, need to add this module to the `composer.json` autoload rules, and then remove the method entirely.